lib/functions.js

```diff
@@ -626,6 +626,7 @@ const getNightlyResult = async (date) => {
 async function getConfig() {
     const config = require('../config.json');
     const help = require('./settings-help');
+    const runTimeList = require('./settings-help-run-time-list');
```

Comment:
It might be enough to have a local constant instead, as it is only imported in one place. Unless you plan to split up the code in the future.